### DISCUSSION TITLE
Add suffix to access point

### DISCUSSION THREAD

**Discussion by hackintoshlover** [RaspAP/raspap-webgui#1474] [General]
Hi,
Thanks for such a wonderful project.
Looking for a way to automatically add a sufiix  <AP Name> / 2.4GHz if hotspot is in 2.4GHz mode and  <AP Name> / 5GHz if hotspot is in 5GHz mode.

**Comment by billz** [comment]
The function below can be modified to append a suffix to the `txtssid` field:

https://github.com/RaspAP/raspap-webgui/blob/ff7f1f01b406f16c2ca810553b57d523f9dcc145/app/js/custom.js#L504-L510
